go/vt/automation/testing_emit_echo_task.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015, Google Inc. All rights reserved.
```

Comment:
sorry I mean "_test" suffix, I believe it is Go's convention: Files with the suffix _test.go are only compiled and run by the go test tool. 

http://stackoverflow.com/questions/25161774/what-are-conventions-for-filenames-in-go